tests/unit/transformers/test_base.py

```diff
@@ -1272,3 +1272,224 @@ def _reverse_transform(self, data):
+        """Test the ``_get_prefix`` method."""
+        # Setup
+        transformer = BaseMultiColumnTransformer()
+
+        # Run and Assert
+        with pytest.raises(NotImplementedError):
+            transformer._get_prefix()
+
+    def test__get_output_to_property(self):
+        """Test the ``_get_output_to_property`` method."""
+        # Setup
+        transformer = BaseMultiColumnTransformer()
+        transformer.output_properties = {
+            'col_1': {'sdtype': 'numerical'},
+            'col_2': {'sdtype': 'categorical'},
+            'col_3': {'next_transformer': None},
+        }
+        prefixes = {
+            'col_1': 'prefix_1',
+            'col_2': 'prefix_2',
+            'col_3': 'prefix_3',
+        }
+        transformer._get_prefix = Mock(return_value=prefixes)
+
+        # Run
+        output = transformer._get_output_to_property('sdtype')
+
+        # Assert
+        expected_output = {
+            'prefix_1.col_1': 'numerical',
+            'prefix_2.col_2': 'categorical',
+        }
+        assert output == expected_output
+        transformer._get_prefix.assert_called_once_with()
+
+    def test__get_output_to_property_with_single_prefix(self):
+        """Test the ``_get_output_to_property`` method when the prefix is a single string."""
+        # Setup
+        transformer = BaseMultiColumnTransformer()
+        transformer.output_properties = {
+            'col_1': {'sdtype': 'numerical'},
+            'col_2': {'sdtype': 'categorical'},
+            'col_3': {'sdtype': 'boolean'},
+        }
+        prefixes = 'prefix'
+        transformer._get_prefix = Mock(return_value=prefixes)
+
+        # Run
+        output = transformer._get_output_to_property('sdtype')
+
+        # Assert
+        expected_output = {
+            'prefix.col_1': 'numerical',
+            'prefix.col_2': 'categorical',
+            'prefix.col_3': 'boolean',
+        }
+        assert output == expected_output
+        transformer._get_prefix.assert_called_once_with()
+
+    def test__get_output_to_property_with_prefix_none(self):
+        """Test the ``_get_output_to_property`` method when the prefix is None."""
+        # Setup
+        transformer = BaseMultiColumnTransformer()
+        transformer.output_properties = {
+            'col_1': {'sdtype': 'numerical'},
+            'col_2': {'sdtype': 'categorical'},
+            'col_3': {'sdtype': 'boolean'},
+        }
+        prefixes = None
+        transformer._get_prefix = Mock(return_value=prefixes)
+
+        # Run
+        output = transformer._get_output_to_property('sdtype')
+
+        # Assert
+        expected_output = {
+            'col_1': 'numerical',
+            'col_2': 'categorical',
+            'col_3': 'boolean',
+        }
+        assert output == expected_output
+        transformer._get_prefix.assert_called_once_with()
+
+    def test__validate_columns_to_sdtypes(self):
+        """Test the ``_validate_columns_to_sdtypes`` method."""
```

Comment:
nitpick: for these descriptions I think it's more useful to say what's being tested. eg
```
Test that an error with the missing column names is raised.
```